tox.ini

```diff
@@ -88,7 +88,7 @@ deps = {[with-debug-tools]deps}
        jsonschema
 
 [testenv:py3_debug]
-basepython = python3.6
+basepython = python3.7
```

Comment:
Good catch, I must have missed these :+1: 

While we are at it, could we also change the new `check_vendored` environment which was developed on a parallel track?